1. controllers/articles.controller.js

```diff
@@ -0,0 +1,20 @@
+const { selectArticles } = require("../2. models/articles.model");
+
+exports.getArticles = (req, res, next) => {
+  const { article_id } = req.query;
+
+  if (!article_id || isNaN(article_id)) {
+    return res.status(400).send({ msg: "Bad Request" });
+  }
```

Comment:
Your controller should not be responsible for handling errors. This can and should also be handled by psql.

Reply:
Completed